core/mavlink_system.h

```diff
@@ -172,6 +177,20 @@ class MAVLinkSystem
     static void system_thread(MAVLinkSystem *self);
     static void send_heartbeat(MAVLinkSystem &self);
 
+    // FIXME: Is the method name OK ? OR Should it be make_command_for_flight_mode(...) ?
```

Comment:
This should be fixed before we merge this PR, right?

Reply:
Yes, I wanted to get feedback about naming. :)